internal/services/storage/client/data_plane.go

```diff
@@ -37,6 +37,13 @@ func (Client) DataPlaneOperationSupportingAnyAuthMethod() DataPlaneOperation {
 	}
 }
 
+func (Client) DataPlaneOperationSupportingNoSharedKeyAuth() DataPlaneOperation {
```

Comment:
If we do keep this helper, it should rather be called:

```suggestion
func (Client) DataPlaneOperationSupportingOnlyAadAuth() DataPlaneOperation {
```

Reply:
Thx